com.github.sakya.corechess.json

```
@@ -0,0 +1,64 @@
+{
+  "app-id": "com.github.sakya.corechess",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "21.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.dotnet6"
+  ],
+  "command": "CoreChess",
+  "finish-args": [
+    "--filesystem=host",
+    "--socket=x11",
+    "--socket=wayland",
```

```suggestion
    "--socket=fallback-x11",
    "--socket=wayland",
```

if it supports wayland.